collect_app/src/debug/AndroidManifest.xml

```diff
@@ -6,7 +6,7 @@
     <uses-permission android:name="android.permission.DISABLE_KEYGUARD"/>
 
     <!-- Allows changing locales -->
-    <uses-permission android:name="android.permission.CHANGE_CONFIGURATION" />
+    <uses-permission android:name="android.permission.CHANGE_CONFIGURATION" tools:ignore="ProtectedPermissions" />
```

Comment:
Just for others to know what this is for. CHANGE_CONFIGURATION can only be granted to system apps. Since this is a debug build, we are OK to use this permission, but we do have to tell lint to ignore it.